whatsapp_fn.py

```diff
@@ -41,6 +44,7 @@ def get_auth_key():
 
 
 def send_whatsapp_message(url,to,message,preview_url=False):
+    add_message(to, bot_number, to, message, "text", "whatsapp", "outgoing")
```

Comment:
We don't need the `outgoing` tag since each the direction will be uniquely identified by `from` and `to`. What do you think ?

Reply:
Yes, this was a feature ask from @sidPhoenix17 as this tag will come handy to the person auditing the log.

Reply:
@satishck1992 as discusses over call I am marking this comment as not relevant now.